redux/auth/reducer.ts

```diff
@@ -61,7 +76,7 @@ function reducer(
     }
 
     case LOGIN_FAILURE:
-      Alert.alert('Something went wrong', action.payload);
+      if (action.payload) Alert.alert('Something went wrong', action.payload);
```

Comment:
In my opinion the alerts should not happen in reducer. I would move it either to saga or to the component (in `useEffect`).
What do you think?

Reply:
I also don't like it. thx for inspiration :smile:  I will deal with it, I have an idea